### DISCUSSION TITLE
Welcome to solarflow-control Discussions!

### DISCUSSION THREAD

**Discussion by reinhard-brandstaedter** [reinhard-brandstaedter/solarflow-control#81] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by holgerlamm** [comment]
Dear Reinhard,

very interesting to read about the project here. I would like to try the Solarflow-control, but i am using a Synolgy NAS for maintaining my dockers. Unfortunately, I have been searching in vain for instructions on how to set up solarflow-control on a Synology NAS. I am particularly interested in the settings for setting up the container on the NAS. You can make a lot of mistakes here and waste a lot of time. I would therefore be very interested in some help.

Thank you very much!
